packages/language-server/src/util/toLS.ts

```diff
@@ -196,17 +195,19 @@ export function hover(hover: core.Hover, doc: TextDocument): ls.Hover {
 export function inlayHint(
 	hint: core.InlayHint,
 	doc: TextDocument,
-): MyLspInlayHint {
+): ls.InlayHint {
 	return {
 		position: doc.positionAt(hint.offset),
-		text: hint.text,
+		label: hint.label,
+		...hint.paddingLeft ? { paddingLeft: hint.paddingLeft } : {},
+		...hint.paddingRight ? { paddingRight: hint.paddingRight } : {},
```

Comment:
is this any different than just returning the `paddingLeft` (or right) property directly?

```suggestion
		paddingLeft: hint.paddingLeft,
		paddingRight: hint.paddingRight,
```

as it is now with the spread property is kinda hard to read in comparison, assuming they're the same functionally (which I think they are?)

Reply:
probably the same functionality. i just didn't want to have extra keys whose value is `undefined` (e.g. `{ label: 'hint', paddingLeft: undefined, paddingRight: true }` v.s. `{ label: 'hint', paddingRight: true }`).

Reply:
actually I'm pretty sure an `undefined` property won't actually show up in an object.

i.e.

```ts
const x = { foo: undefined, bar: 2 }
console.log(x)
// { bar: 2 }
```